src/Limiters/quasimonotone.jl

```diff
@@ -36,49 +36,38 @@ Then call [`apply_limiter!`](@ref) on the output fields:
 
     apply_limiter!(ρq, ρ, limiter)
 """
-struct QuasiMonotoneLimiter{D, G}
+struct QuasiMonotoneLimiter{D, G, FT}
     "contains the min and max of each element"
     q_bounds::D
     "contains the min and max of each element and its neighbors"
     q_bounds_nbr::D
     "communication buffer"
     ghost_buffer::G
+    "relative tolerance for tracer mass change"
+    rtol::FT
 end
 
 
-function QuasiMonotoneLimiter(ρq::Fields.Field, ρ::Fields.Field)
-    data = Fields.field_values(ρq)
-    topology = Spaces.topology(axes(ρq))
-    QuasiMonotoneLimiter(data, topology)
+function QuasiMonotoneLimiter(ρq::Fields.Field, rtol = eps(eltype(parent(ρq))))
```

Comment:
Maybe make `rtol` a keyword argument (that's how it is used elsewhere).

Also add a deprecation for the modified signature.

Reply:
Just to clarify, you mean I should write `Base.@deprecate QuasiMonotoneLimiter(ρq::Fields.Field, ρ::Fields.Field) QuasiMonotoneLimiter(ρq::Fields.Field; rtol = eps(eltype(parent(ρq))))`?